bin/build-release.sh

```diff
@@ -109,7 +109,11 @@ function package {(
 )}
 
 function dockerImage {(
-  cmd="docker build -t mesos/storm:git-`git rev-parse --short HEAD` ."
+  cmd="docker build \
+       --build-arg MESOS_RELEASE=$MESOS_RELEASE \
+       --build-arg STORM_RELEASE=$STORM_RELEASE \
+       --build-arg MIRROR=$MIRROR \
+        -t mesos/storm:git-`git rev-parse --short HEAD` ."
```

Comment:
Would be cool here to embed the storm/mesos versions into the tag (possibly in the version).  Then it'd be trivial to release a few different images on dockerhub.  Related to #18, #83.